src/core/declarativewebcontainer.h

```diff
@@ -70,7 +70,7 @@ class DeclarativeWebContainer : public QWindow, public QQmlParserStatus, protect
     Q_PROPERTY(bool privateMode READ privateMode WRITE setPrivateMode NOTIFY privateModeChanged FINAL)
     Q_PROPERTY(bool activeTabRendered READ activeTabRendered NOTIFY activeTabRenderedChanged FINAL)
 
-    Q_PROPERTY(QQmlComponent* webPageComponent MEMBER m_webPageComponent NOTIFY webPageComponentChanged FINAL)
+    Q_PROPERTY(QQmlComponent* webPageComponent MEMBER m_webPageComponent WRITE setWebPageComponent NOTIFY webPageComponentChanged FINAL)
```

Comment:
Even thou above is a valid property declaration, I'd change ```MEMBER m_webPageComponent``` to ```READ webPageComponent``` and add getter.